cns/service/main.go

```diff
@@ -579,6 +579,13 @@ func main() {
 			logger.Errorf("Failed to start CRD Controller, err:%v.\n", err)
 			return
 		}
+
+		// Setting the remote ARP MAC address to 12-34-56-78-9a-bc on windows for external traffic
+		err = platform.SetSdnRemoteArpMacAddress()
```

Comment:
You need to set this regKey when ChannelModel == MultiTenantCRD as well.

Reply:
added for multitenant scenario